scripts/publish_docs.sh

```diff
@@ -0,0 +1,54 @@
+make
+if [[ $APIS_EMAIL != "" ]]
+then
+  git config --global user.email "${APIS_EMAIL}"
+  git config --global user.name "${APIS_USERNAME}"
+  tag_name="${GITHUB_REF##*/}"
+  LOCAL=0
+else
+  tag_name="local_test"
+  LOCAL=1
+fi
+
+rm -rf generated_docs docs
+mkdir generated_docs
+
+if [[ $LOCAL == 1 ]]
```

Comment:
You don't necessarily need this flag, you can make it dependent on whether or not the `GITHUB_REF` or another environment variable set by GitHub actions is set.